test/token/ERC721/ERC721.behavior.js

```diff
@@ -328,7 +329,7 @@ function shouldBehaveLikeERC721 (errorPrefix, owner, newOwner, approved, another
 
         describe('to a contract that does not implement the required function', function () {
           it('reverts', async function () {
-            const nonReceiver = this.token;
```

Comment:
Why are we not using the same token deployed from `ERC721Wrapper.test.js` here and deploying a new token instead?

Reply:
The behavioral tests assume that `this.token` is not an ERC721 receiver, however, the wrapper is required to be a receiver so this assumption breaks.

Here I'm creating any nonreceiver contract just to fix it. By default I'm creating an ERC721 to keep the past assumption consistent with previous tests.

Reply:
Ah, I understand.